spec/components/search_spec.rb

```diff
@@ -100,7 +100,7 @@ def component_name
   it "applies data attributes when provided" do
     render_component(
       button_text: "Some test text",
-      data_attributes: {
+      button_data_attributes: {
```

Comment:
If you're certain that these button attributes aren't set anywhere, I think it'd be a good opportunity to remove this option. I'd double check it's not being picked up in any JS anywhere.